tls/s2n_ktls_parameters.h

```diff
@@ -41,6 +41,8 @@
     #define S2N_TLS_TX 1
     #define S2N_TLS_RX 2
 
+    #define S2N_TLS_SET_RECORD_TYPE TLS_SET_RECORD_TYPE
```

Comment:
Could you explain to me what these two macros are for? Because you seem to use TLS_SET_RECORD_TYPE for the get_control_data function and the TLS_GET_RECORD_TYPE for the set_control_data function. So I'm a little confused.

Reply:
Hehe yea pretty confusing because its only usage is in the mock implementation. Here is the usage in the next PR where the set/get align better: https://github.com/aws/s2n-tls/pull/4147/commits/7917d2b855b9ab963a147e65a147ab83847c2369#diff-732fdef032104706c8a36a45efd7c4098fed24bcec6906b78c81b1bfc9e61eaeR203

Reply:
Added a comment to where they are used in the mock https://github.com/aws/s2n-tls/pull/4146/commits/1b2f9486e7c6c5dbc82f2ba1a75d15cf5cb4c5df